tests/e2e-server-upgrade/online-upgrade-kill-transient/setup-vdb/base/setup-vdb.yaml

```
@@ -11,31 +11,32 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-apiVersion: vertica.com/v1beta1
+apiVersion: vertica.com/v1
 kind: VerticaDB
 metadata:
   name: v-base-upgrade
+  annotations:
+    vertica.com/include-uid-in-path: "true"
+    vertica.com/k-safety: "0"
+    vertica.com/requeue-time: 4 
 spec:
   image: kustomize-vertica-image
-  communal:
-    includeUIDInPath: true
+  communal: {}
   local:
     requestSize: 100Mi
-  kSafety: "0"
   upgradePolicy: ReadOnlyOnline
   subclusters:
     - name: pri
       size: 1
-      isPrimary: true
+      type: primary
     - name: sec
       size: 1
-      isPrimary: false
+      type: secondary
   temporarySubclusterRouting:
     template:
       name: transient
```

Haven't tested this yet, but if it failed, try to add type as secondary in the template:
```suggestion
      name: transient
      type: secondary
```